arch/arm/src/stm32h7/stm32_fdcan_sock.c

```diff
@@ -2080,7 +2080,7 @@ int fdcan_initialize(struct fdcan_driver_s *priv)
     }
 
 #ifdef CONFIG_STM32H7_FDCAN_REGDEBUG
-  const fdcan_bitseg *tim = &priv->arbi_timing;
+  struct fdcan_bitseg *tim = &priv->arbi_timing;
```

Comment:
Why `const` is dropped?

Reply:
The tim variable is used in other locals and the same funtion

Reply:
but you can change const pointer to another object.